jumpscale/clients/stellar/__init__.py

```diff
@@ -51,45 +49,6 @@ def check_stellar_service(self):
 
         return services_status
 
-    def create_testnet_funded_wallet(self, name: str) -> bool:
```

Comment:
I think this method is used a lot, for example in admin dashboard and the unit tests, changing it's location/signature may lead to break these things ?

Reply:
The admin dashboard uses the wallet actor to create a new account and only the threebot should fund its testnet wallet. If this breaks a test, we should revise the test.